docs/reference/observability/metrics.md

```diff
@@ -0,0 +1,46 @@
+# Crossplane Metrics
```

Comment:
I'd prefer to document our metrics on https://docs.crossplane.io, which lives in https://github.com/crossplane/docs.